cockatrice/src/tab_deck_editor.cpp

```diff
@@ -45,6 +46,10 @@
 #include "cardframe.h"
 #include "filterbuilder.h"
 
+const QStringList TabDeckEditor::fileNameFilters = QStringList()
+    << QObject::tr("Cockatrice set format (*.xml)")
```

Comment:
There is no "set format" or sth, we just have a card databases...
People can import a xml database with cards from a whole block or any other combination of cards/sets as well.
(whatever kind of cards/sets are in there it needs to be a xml file)

`Cockatrice card database (*.xml)` might be more general and better suitable.


<br>
Thinking further... should the menu say something like `Add custom set(s) or card(s)` or more simple `Add custom sets/cards` ?